CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ### Fixed
 - Fix version and build ([#254](https://github.com/opensearch-project/opensearch-java/pull/254))
+- Support weight function in function score query ([#880](https://github.com/opensearch-project/opensearch-java/pull/880))
```

Comment:
Assuming we get the fix into 2.x, we don't need to include it in the "Unreleased 3.0" part. (It's just for changes that won't be backported to 2.x because they're breaking.)

Reply:
Thanks, removed 3.0 changelog entry